src/App.tsx

```diff
@@ -1,13 +1,16 @@
-import React, { useContext } from "react";
+import React from "react";
 import "./App.css";
-import { AppContext } from "./components/AppContext/AppContext";
+import { IngredientSearch } from "./components/IngredientSearch/IngredientSearch";
 
-export const App: React.FC = () => {
-  const { data } = useContext(AppContext);
-
-  const images = data.map(({ image, name, id }) => (
-    <img key={id} src={image} alt={name} />
-  ));
+const PageStyles: React.CSSProperties = {
+  margin: "0 auto",
+  padding: "2rem",
+};
 
-  return <div className="App">{images}</div>;
+export const App: React.FC = () => {
```

Comment:
Sorry for being annoying about naming but I'm thinking I want to rename this file because it's not clear to my brain. 

We can move it to a discussion if you can't think of anything off of hand (bc I can't haha).

Reply:
Moved it to an issue #14